client/hosting/server-settings/components/defensive-mode-card/style.scss

```diff
@@ -31,6 +31,24 @@
 	width: 10px;
 }
 
+.defensive-mode-card__a11n-description {
```

Comment:
We need to set the text color to `--studio-wordpress-blue-80` here.

This color should be applied to the icon inside the banner too.